### PR TITLE
Arnah

### PR DESCRIPTION
**Pull request by Ny-Rianala**
Hey,

I added a new file, please have a look and let me know if something went wrong.

Thanks.
Kati

### CODE REVIEW

File: a/arnah.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Arnah
+
+
+![Arnah](arnah.jpg)
```
Comment (ndorfin) [inline a/arnah.md:4]: This is lowercase, correct.
But the filename of the image has a capital letter!

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — CHANGES_REQUESTED** [review]
The image filename needs to use a lowercase 'a' please.

**Comment by Ny-Rianala** [comment]
Hey,
I fixed the image filename, please have a look at it and let me know if there are things which I need to change.
Thanks,
Kati

**Review by ndorfin — CHANGES_REQUESTED** [review]
Needs a markdown file

**Review by ndorfin — APPROVED** [review]
Well done!

**Review by ndorfin — APPROVED** [review]

**Pull request merged**